scripts/postinstall.ts

```diff
@@ -31,26 +35,18 @@ export function fixFormatter() {
   const formatPath = path.join(process.cwd(), '/../..', '@nrwl/schematics/src/command-line/format.js');
   let formatContent = fs.readFileSync(formatPath, 'UTF-8');
 
-  // Old patch line, globs should've been wrapped in quotes.
-  const oldPatchLine = `    patterns.push("xplat\/**\/*.ts", "!apps/**/platforms/{android,ios}/**/*.ts");`;
-  const patchLine = `    patterns.push('\"xplat\/**\/*.ts\"', '\"!apps/**/platforms/{android,ios}/**/*.ts\"');`;
-  if (formatContent.indexOf(oldPatchLine) !== -1) {
-    console.log('Old patch for nx format have been applied, replace with new patch');
-    fs.writeFileSync(formatPath, formatContent.replace(oldPatchLine, patchLine));
-    return;
-  }
-
+  const patchLine = `    // PATCHED by @nstudio/xplat\n    patterns.push('"xplat/**/*.ts"', '"!apps/**/platforms/{android,ios}/**/*"');`;
```

Comment:
I had issues the last time this patch line was modified - can you confirm that this patch works properly in a fresh workspace based off latest Nx?

Thanks for this PR @m-abs

Reply:
Sounds like you did but just want to be extra sure 🤗

Reply:
I tested it on a fresh project.  But I'd really like an automated way of testing it.

Reply:
That would be nice :) thanks again I’ll merge and test this locally befor bumping a release 👍